src/components/CloseButton/CloseButton.css

```diff
@@ -0,0 +1,38 @@
+@layer components {
+	.close-button {
+		cursor: pointer;
+		width: 16px;
```

Comment:
Pleaes utilize REM values and when possible utilize our system design variables.